hoomd/hpmc/update.py

```diff
@@ -845,11 +848,25 @@ class QuickCompress(Updater):
     box move sizes will be uniformly distributed between ``min_scale`` and 1.0
     (with no consideration of ``min_move_size``).
 
+    When using a `BoxVariant` for `target_box`, `complete` returns `True` if the
+    current simulation box is equal to the box corresponding to `target_box`
+    evaluated at the current timestep and there are no overlaps in the system.
+    To ensure the updater has compressed the system to the final target box, use
+    a condition that checks both the `complete` attribute of the updater and the
+    simulation timestep. For example::
+
+        target_box = hoomd.variant.box.InverseVolumeRamp(
+            sim.state.box, sim.state.box.volume / 2, 0, 10000)
+        qc = hoomd.hpmc.update.QuickCompress(10, target_box)
+        while (not qc.complete) and sim.timestep < qc.target_box.t_ramp:
+            sim.run(100)
```

Comment:
```suggestion
        while sim.timestep < target_box.t_ramp + target_box.t_start or (not qc.complete):
            sim.run(100)
```
Use `or` to continue running past the end of `t_start + t_ramp` until the compression is complete. Also change the order of the lazy evaluation to make it more obvious this while loop is until the end of the ramp.